leapp/repository/__init__.py

```diff
@@ -145,7 +145,7 @@ def load(self, resolve=True, stage=None):
             sys.meta_path.append(LeappLibrariesFinder(module_prefix='leapp.libraries.common', paths=self.libraries))
             sys.meta_path.append(LeappLibrariesFinder(module_prefix='leapp.workflows.api', paths=self.apis))
 
-        if not stage or stage is _LoadStage.ACTORS:
+        if not stage or stage is _LoadStage.ACTORS and not skip_actors_discovery:
```

Comment:
```suggestion
        if not skip_actors_discovery:
             if not stage or stage is _LoadStage.ACTORS:
```

Reply:
Accepting the suggestion. Could you please explain why do you think it is better?